### PR TITLE
support for exception rules #1580

### PR DESCRIPTION
**Pull request by Sriramkashyapks**

### CODE REVIEW

**Comment by Sriramkashyapks** [comment]
Hey there if the above code Ive pushed is correct please merge it

**Comment by matthyx** [comment]
@Sriramkashyapks I don't see any code change in your commit... also please don't forget test coverage as well :)

**Comment by nunidoron** [comment]
As @matthyx said, please dont forget to cover your tests :)

File: examples/exceptions/exclude-allowed-hostPath-control.json [hunk 1]
```diff
@@ -1,22 +1,35 @@
-[
-    {
+{
+    "account": "...",
+    "excludedNamespaces": ["..."],
+    "failThreshold": "...",
+    "complianceThreshold": "...",
+    "format": "...",
```
Comment (nunidoron) [inline examples/exceptions/exclude-allowed-hostPath-control.json:6]: Are sure that you need it?

**Comment by matthyx** [comment]
@Sriramkashyapks can you have a look at the review?

**Pull request closed without merging**